install/kubernetes/helm/istio/values.yaml

```diff
@@ -412,7 +412,7 @@ global:
   # The namespace where globally shared configurations should be present.
   # DestinationRules that apply to the entire mesh (e.g., enabling mTLS),
   # default Sidecar configs, etc. should be added to this namespace.
-  # configRootNamespace: istio-config
+  configRootNamespace: istio-config
```

Comment:
You have to move the default destination rules from auth into this namespace.. and thats where the helm problems start.

Reply:
Hmmm. the tests are passing. You think there are things that are being missed? 

I will look at removing the hardcoded value in DefaultMeshConfig next.

Reply:
Now that’s a mystery. I did this before 1.1 and hit issues.

Reply:
>You have to move the default destination rules from auth into this namespace.. 

What do u mean by `default destination rules`? I donot think we have global DRs now.

Reply:
oh, ic , it's the mTLS one

Reply:
@mbanikazemi look at install/kubernetes/helm/istio/charts/security/templates/enable-mesh-mtls.yaml

Reply:
@hzxuzhonghu See the latest patch please. Is this the correct approach?